pkg/asset/installconfig/platform.go

```diff
@@ -59,9 +62,7 @@ var (
 // Platform is an asset that queries the user for the platform on which to install
 // the cluster.
 type platform struct {
-	AWS       *types.AWSPlatform
-	Openstack *types.OpenStackPlatform
-	Libvirt   *types.LibvirtPlatform
+	platform types.Platform
```

Comment:
why not `type platform types.Platform` ?

Reply:
> why not `type platform types.Platform`?

:+1:  Fixed with 634165f -> 8426554 (which also fixes a godoc and some unit tests).